config/.env

```diff
@@ -32,7 +32,7 @@ MGMT_BACKEND_PRIVATEPORT=3084
 # artifact-backend
 ARTIFACT_BACKEND_HOST=artifact-backend
 ARTIFACT_BACKEND_PUBLICPORT=8085
-ARTIFACT_BACKEND_PRIVATEPORT=3085
+ARTIFACT_BACKEND_PRIVATEPORT=8085
```

Comment:
@donch1989 @heiruwu when I called the private API with the private port the gateway couldn't reach the service. Could you have an 👀 [on the `artifact-backend` config](https://github.com/instill-ai/artifact-backend/blob/main/Makefile#L20) to check if I'm missing something to serve the private API on the private port?

Reply:
the port forward flag with docker command is to forward the port onto host machine, we actually don't need to forward the private port. api-gateway is using the internal docker `instill-network` to connect to artifact-backend

Reply:
you will need to add the artifact related env variables [here](https://github.com/instill-ai/instill-core/blob/main/docker-compose.yml#L27-L45), since the default value will be overwrite by `.env.envsubst`

Reply:
Fixed and tested against this branch ✅  https://github.com/instill-ai/artifact-backend/pull/9